app/assets/stylesheets/restaurants.scss

```diff
@@ -1,3 +1,112 @@
-// Place all the styles related to the Restaurants controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin make-float-boxes(){
+  float: left;
+  display: inline-block;
+}
+
+@mixin image-no-repeat($url){
+  background-image: url($url);
+  background-repeat: no-repeat;
+
+}
+.header{
+  position: relative;
+  width: 100%;
+  height: 200px;
+  background-size: cover;
+  @include image-no-repeat('header.png');
+
+  .header-text {
+    position: absolute;
+    width: 100%;
+    height: 50%;
+    top: 25%;
+    background-position: center;
+    @include image-no-repeat('header-text.png');
+  }
+
+}
+
+.list{
+
+  .filters-box{
+    position: relative;
+    width: 100%;
+    height: 150px;
+    background-color: cornflowerblue;
+
+    .filters-wrapper{
+      position: relative;
+      width:75%;
+      height: 80%;
+      top: 20px;
+      margin: auto;
+    }
+    .filter-item{
+      position: relative;
+      width: 30%;
+      height: 100px;
+      background-color: #2E2F30;
```

Comment:
the best practice for colors would be to declare them at the top of the file or in a customization CSS files, and then re-use then, i.e `$primary-color = #333`, and then `.btn {background-color: $primary-color`, while avoiding hardcoded color values in the CSS.